btree/btree.c

```diff
@@ -1672,9 +1692,16 @@ static int64_t node_get(struct node_op *op, struct td *tree,
  *
  * @return next state
  */
-static void node_put(struct nd *node){
+static void node_put(struct nd *node, struct m0_be_tx *tx){
+	struct node_op op;
```

Comment:
Pass `struct node_op *op` to this function (similar to node_get()) and use that.

Reply:
made this changes.